pulp-in-one-container.md

```diff
@@ -41,7 +39,7 @@ $ podman run --detach \
              --volume "$(pwd)/pgsql":/var/lib/pgsql:Z \
              --volume "$(pwd)/containers":/var/lib/containers:Z \
              --device /dev/fuse \
-             quay.io/pulp/pulp
+             pulp/pulp
```

Comment:
Why not force the image being downloaded from Quay?

Reply:
1. It simplifies the command.
2. It makes the command more robust to be able to access any 3 of the registries.
3. We did this in the past because the image used to only be pushed to the quay.io registry.